### ISSUE TITLE
Set up User Roles

### ISSUE THREAD

**Issue by esoterik** [YaleSTC/vesta#19]
We figure out what we want to use to manage permissions (cancancan or pundit).

Also, we should set up user roles as enums!

Blocked on #5.

**Comment by esoterik** [comment]
I've removed the permission-central aspects of the data entry from those respective issues; this issue should probably implement the restrictions that we want (or we should open up a new issue for that when the time comes)

**Comment by esoterik** [comment]
I definitely think that we should use [pundit](https://github.com/elabs/pundit) over cancancan. It's resource-centric; instead of creating a big `Ability` class, permissions are defined in a `Policy` class for each resource, which I think will be much cleaner in the long run.

**Comment by orenyk** [comment]
We will install pundit, set up an enum for user roles, and authorize only admins to create new users.

**Comment by orenyk** [comment]
asdas
